doc/pprof.md

```diff
@@ -214,6 +214,10 @@ search for them in a directory pointed to by the environment variable
 * **-weblist= _regex_:** Generates a source/assembly combined annotated listing for
   functions matching *regex*, and starts a web browser to display it.
 
+Samples in a profile may have tags. These tags have a name and a value; this
```

Comment:
This added paragraph sounds a bit out of tune in this "Annotated code" section. Why here?

Reply:
Made an error when moving docs around. Fixed.